composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ricbra/php-discogs-api",
+    "name": "davalb/php-discogs-api",
```

Comment:
Could you revert this commit? As it would rename this library on packagist.

If you want to locally install your own fork you shouldn't add it to packagist; there's an more easy way which you can configure in <code>composer.json</code>. If you want I can help you with that!